bin/worker

```diff
@@ -5,6 +5,14 @@ if (process.env.NEWRELIC_KEY) {
   require('newrelic')
 }
 
+const Sentry = require('@sentry/node')
+
+Sentry.init({
```

Comment:
I've copy/paste this for now to ensure it isn't called twice. This can't go in `lib/worker/index.js` because we require that from within the web process.

I like the idea of extracting this out in the future.